Controller/TreeController.php

```diff
@@ -41,20 +43,35 @@ class TreeController extends Controller
     private $confirmMove = false;
 
     /**
+     * @var \PHPCR\SessionInterface
+     */
+    private $session;
+
+    /**
+     * @param ManagerRegistry $manager
+     * @param string $sessionName
      * @param string $repositoryName
      * @param string $template
-     * @param array  $defaults
-     * @param bool   $confirmMove
+     * @param array $defaults
+     * @param bool $confirmMove
```

Comment:
i am now enabling styleci and will let it make the code according to the dev-kit styleci.yml configuration.